README.md

```diff
@@ -12,10 +12,23 @@
 </p>
 
 > Note: This library is a work in progress until the [new API](https://github.com/nestdotland/api) for nest.land is deployed.
+
 ## yolk
 
 Yolk is nest.land's official API client library for Deno.
 
+### Import with Deno
+
+```shell script
+import * as yolk from "https://x.nest.land/yolk@0.1.0-rc4/mod.ts"
+```
+
+### Use with Node
```

Comment:
We are not supporting Node yet because the `fetch` does not exist there. For now, change `Node` to `Web`

Reply:
Apologies! Completely forgot about that. Thanks for reminding me.